doc/cephfs/dirfrags.rst

```diff
@@ -79,16 +80,16 @@ operations on directory fragments.  The decaying load counters have an
 exponential decay based on the ``mds_decay_halflife`` setting.
 
 On writes, the write counter is
-incremented, and compared with ``mds_bal_split_wr``, triggering a 
+incremented, and compared with ``mds_bal_split_wr``, triggering a
 split if the threshold is exceeded.  Write operations include metadata IO
-such as renames, unlinks and creations. 
+such as renames, unlinks and creations.
 
 The ``mds_bal_split_rd`` threshold is applied based on the read operation
 load counter, which tracks readdir operations.
 
-By the default, the read threshold is 25000 and the write threshold is
-10000, i.e. 2.5x as many reads as writes would be required to trigger
-a split.
+By the default, the read threshold is 25000 omap entries and the write
```

Comment:
Isn't  the threshold here represents the number of reads/writes operations rather than omap entries?

Reply:
reads/writes on omap entries right?

Reply:
As mentioned in line 84, the write operations include rename, unlinks, creates, so whenever mds sees these operations from clients, it increments the associated decay counter. Hence the max value is also of the same unit i.e., the number of write operations ?

Reply:
Right, maybe I can remove the changes here, what say? @kotreshhr

Reply:
@vshankar Thanks!